packages/graphql/src/modules/user/infra/mongoose/repositories/mongoUsersRepository.ts

```diff
@@ -1,10 +1,13 @@
+import DataLoader from 'dataloader';
 import {v4 as uuid} from 'uuid';
 
 import User from '../../../domain/User';
 import UsersRepository, {FindUserByEmailOrProviderId} from '../../../domain/UsersRepository';
 import UserModel, {IUserDocument} from '../models/UserModel';
 
 export default function mongoUsersRepository(): UsersRepository {
+  const userLoader = new DataLoader<string, User>(userIds => UserModel.find({id: {$in: userIds}}));
```

Comment:
Cool, I think it's enough. But we had a `UserLoader` class with some methods. I don't know how useful they are,  @renanmav could you clarify? [UserLoader](https://github.com/animavita/animavita/blob/next/packages/graphql/src/modules/user/presentation/routes/queries/UserLoader.ts)

Reply:
As far as I know, those methods weren't used anywhere. Maybe @renanmav had some plans for them but I don't think they were being used.

Reply:
Yeah, I don't think they are being used